src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Client.UnitTests/CommitmentsApiClient/WhenCallingTheEndpoints.cs

```diff
@@ -197,24 +196,99 @@ public async Task DeleteCohort_VerifyUrlAndDataIsCorrectPassedIn()
         }
 
         [Test]
-        public async Task GetApprovedApprenticeships_VerifyUrlAndDataIsCorrectPassedIn()
+        public async Task GetApprenticeships_VerifyUrlAndDataIsCorrectPassedIn()
         {
             //Arrange
             var expectedProviderId = (uint) 10;
 
             //Act
-            await _fixture.CommitmentsApiClient.GetApprovedApprenticeships(expectedProviderId);
+            await _fixture.CommitmentsApiClient.GetApprenticeships(expectedProviderId);
 
             //Assert$
-            _fixture.MockRestHttpClient.Verify(x => x.Get<IEnumerable<ApprenticeshipDetails>>($"api/Apprentices/{expectedProviderId}", null, CancellationToken.None));
+            _fixture.MockRestHttpClient.Verify(x => x.Get<GetApprenticeshipsResponse>($"api/apprenticeships/{expectedProviderId}?sortField=&reverseSort=False", null, CancellationToken.None));
         }
-		
+
+        [Test]
+        public async Task GetApprenticeships_VerifyUrlAndPageNumberIsCorrectPassedIn()
+        {
+            //Arrange
+            var expectedProviderId = (uint) 10;
```

Comment:
We model providerId everwhere else as a long

Reply:
The ukprn should be no bigger than an int from what we have been told in the past (the making it a long was an historical mistake unless things have changed). I made it a uint as its never going to be negative but i'll change it to make it consistent. Its worth noting the above though for future reference as it being stored as a long is probably overkill moving forward